pkg/scheduler/extender/extender.go

```diff
@@ -431,6 +436,39 @@ func (e *Extender) createReservation(ctx context.Context, namespace string, name
 		// cannot create reservation
 		return err
 	}
+	// wait for reservation to be confirmed
+	// continue on reading error
+	// return after reservation confirmed
+	// or wait context deadline
+	readCh := make(chan struct{})
+	ll := e.logger.WithFields(logrus.Fields{
+		"method":      "createReservation",
+		"sessionUUID": ctx.Value(base.RequestUUID),
+	})
+	go func() {
+		for _, timer := range []time.Duration{1 * time.Second, 3 * time.Second, 10 * time.Second} {
```

Comment:
If we change 15 seconds http timeout to something else will this code work correctly. May be it make sense to start with 1 second and multiple by 2 on each iteration and don't limit number of iterations and max value?